app/src/androidTest/java/org/mozilla/fenix/screenshots/DefaultHomeScreenTest.kt

```diff
@@ -44,10 +44,14 @@ class DefaultHomeScreenTest : ScreenshotTest() {
             SystemClock.sleep(TestAssetHelper.waitingTimeShort)
             Screengrab.screenshot("HomeScreenRobot_home-screen")
         }.openThreeDotMenu {
-        }.openSettings { }
+        }.openSettings {
+        }.openPrivateBrowsingSubMenu {
```

Comment:
If we want to have a screenshot of this new screen we should add something like... `Screengrab.screenshot("PrivateBrowsingRotob_privateBrowsing")`

@ekager This test suite screenshots is fot that, to get screenshots from the different screens, it is great to add the test here but wondering if this is intended or it should go to the regular tests? Thanks!!